src/homeworks/test_47/index.ts

```diff
@@ -0,0 +1,11 @@
+
+type dataInput = Record<"year" | "month" | "day", number>
+
+const getData = ({year, month, day}: dataInput): Date => new Date(year, month, day)
```

Comment:
Questa funzione non ritorna `Data` ma una `Date` (Data in inglese)
```suggestion
const getDate = ({year, month, day}: dataInput): Date => new Date(year, month, day)
```